multi-node/aws/pkg/config/templates/cluster.yaml

```diff
@@ -1,22 +1,25 @@
 # Unique name of Kubernetes cluster. In order to deploy
 # more than one cluster into the same AWS account, this
 # name must not conflict with an existing cluster.
-# clusterName: kubernetes
+
+clusterName: {{.ClusterName}}
+
+# DNS name routable to the Kubernetes controller nodes
+# from worker nodes and external clients. The deployer
+# is responsible for making this name routable
+externalDNSName: {{.ExternalDNSName}}
+
+### DO NOT CHANGE PARAMETERS ABOVE HERE ###
```

Comment:
Yeah agree. Honestly that is really only relevant if you're going to be attempting to upgrade your cluster, which we are officially not supporting yet for end users. I'll remove it for now.